src/test/java/org/folio/rest/impl/MockServer.java

```diff
@@ -291,6 +293,23 @@ private void handleGetFolioInvoiceNumber(RoutingContext ctx) {
     }
   }
 
+  
+  private void handlePostVoucherStartValue(RoutingContext ctx) {
+    logger.info("got: " + ctx.getBodyAsString());
+    String startValue = ctx.request()
+      .getParam("value");
+    if (startValue.contains(BAD_QUERY)) {
+      serverResponse(ctx, 400, APPLICATION_JSON, Response.Status.BAD_REQUEST.getReasonPhrase());
```

Comment:
/voucher-storage/voucher-number/start/badRequest returns 400 text/plain.
Can be combined with the case for a negative value.

Reply:
Done